File: totaleconomy-api/src/main/java/com/ericgrandt/totaleconomy/model/Sender.java
package com.ericgrandt.totaleconomy.model;

import net.kyori.adventure.text.Component;

public interface Sender {
    /**
     * Sends a message to the sender
     *
     * @param message the {@link Component} that will be sent to the sender
     */
    void sendMessage(Component message);
}

File: totaleconomy-api/src/main/kotlin/com/ericgrandt/totaleconomy/model/Sender.kt
package com.ericgrandt.totaleconomy.model

import net.kyori.adventure.text.Component

interface Sender {
    /**
     * Sends a message to the sender
     *
     * @param message the [Component] that will be sent to the sender
     */
    fun sendMessage(message: Component)
}
